packages/contracts-bedrock/contracts/universal/StandardBridge.sol

```diff
@@ -507,6 +764,36 @@ abstract contract StandardBridge {
         emit ETHBridgeFinalized(_from, _to, _amount, _extraData);
     }
 
+    function _emitBITBridgeInitiated(
```

Comment:
Add comment to describe functionality and parameters.

Reply:
fixed